Tuebingen/UBTaeth2.xml

```diff
@@ -0,0 +1,273 @@
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.8">
+  <locus from="63r" to="86v"/>
+  <title type="complete" ref="LIT6045SerataSegdat"/>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.9">
+  <locus from="86v" to="106v"/>
+  <title type="complete" ref="LIT2097OnHowt"/>
+  <note>Indication for reading: <foreign xml:lang="gez">አመ፡ ፳ወ፱ለመጋቢት፡</foreign></note>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.10">
+  <locus from="106v" to="110r"/>
+  <title type="complete" ref="LIT6048OnPalmSundayRetuaH"/>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.11">
+  <locus from="110r" to="125r"/>
+  <title type="complete" ref="LIT2149OntheW"/>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.12">
+  <locus from="125r" to="139v"/>
+  <title type="complete" ref="LIT2118OntheD"/>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.13">
+  <locus from="139v" to="145v"/>
+  <title type="complete" ref="LIT2109OntheA"/>
+  <note>Indication for reading: <foreign xml:lang="gez">ምንባብ፡ ዘበዓለ፡ ፵፡</foreign></note>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.14">
+  <locus from="145v" to="158"/>
+  <title type="complete" ref="LIT2137OntheP"/>
+  <note>Indication for reading: <foreign xml:lang="gez">ምንባብ፡ ዘበዓለ፡ ፶፡</foreign></note>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.15">
+  <locus from="158v" to="163r"/>
+  <title type="complete" ref="LIT6047On12ApostlesRetuaH"/>
+  <note>Indication for reading: <foreign xml:lang="gez">ምንባብ፡ አመ፭ለሐምሌ፡</foreign></note>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.16">
+  <locus from="163r" to="169v"/>
+  <title type="complete" ref="LIT2145OntheT"/>
+  <note>Indication for reading: <foreign xml:lang="gez">ምንባብ፡ አመ፡ ፲ወ፫፡ ለነሐሴ።</foreign></note>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.17">
+  <locus from="170r" to="189r"/>
+  <title type="complete" ref="LIT1640Homily"/>
+  <note>Indication for reading: <foreign xml:lang="gez">ምንባብ፡ አመ፡ ፲፮፡ ለነሐሴ።</foreign></note>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.18">
+  <locus from="189v" to="196v"/>
+  <title type="complete" ref="LIT2124OntheF"/>
+  <note>Indication for reading: <foreign xml:lang="gez">ምንባብ፡ አመ፡ ፰፡ ለኅዳር፡</foreign></note>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+<colophon xml:id="coloph1" xml:lang="gez">
+<locus target="#197r"/>
+ተፈጸመ፡ መጽሐፈ፡ ርቱዓ፡ ሃይማኖት፡ በወርኃ፡ <sic>ማዝያ፡ </sic> በዘመነ፡ ዮሐንስ፡ በ፲ወ፯፡ ዕለት፡ ወሌሊት፨ በዕለተ፡ በዓለ፡ ስቅለቱ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፨ ዘተወጥነ፡
+በወርኃ፡ ጥር፡ በ፲ወ፪ዕለት፡ ወበ፲ሌሊት፨ በሀገረ፡ <placeName ref="LOC1463Ankoba">አንኮበር</placeName>፡ በብሔረ፡ ሸዋ፡
+</colophon>
+
+  </msItem>
+</msContents>
+
+<physDesc>
+
+  <objectDesc form="Book">
+      <supportDesc>
+          <support>
+              <material key="paper"/>
+          </support>
+          <extent>
+              <measure unit="page">197</measure>
```

Comment:
I would use "leaf" here - they are not 197 pages. Do you prefer to use page because it's made of paper?

Reply:
Corrected, I had no particular intention here.